src/compression_static.rs

```diff
@@ -25,6 +29,29 @@ pub struct CompressedFileVariant<'a> {
     pub extension: &'a str,
 }
 
+/// Initializes static compression.
+pub fn init(enabled: bool, handler_opts: &mut RequestHandlerOpts) {
+    handler_opts.compression_static = enabled;
+    server_info!("compression static: enabled={enabled}");
+}
+
+/// Post-processing to add Vary header if necessary.
+pub(crate) fn post_process(
+    opts: &RequestHandlerOpts,
+    _req: &Request<Body>,
```

Comment:
How about this unused `_req `? I also saw it in `src/security_headers.rs`. Do you have something in mind for it?

Reply:
I merely want the call signatures to be consistent. Right now this is irrelevant of course. But the goal is moving towards calling handlers dynamically – be it via middleware or via some trait.